drivers/gles3/storage/texture_storage.h

```diff
@@ -553,6 +553,7 @@ class TextureStorage : public RendererTextureStorage {
 	void texture_set_data(RID p_texture, const Ref<Image> &p_image, int p_layer = 0);
 	virtual Image::Format texture_get_format(RID p_texture) const override;
 	uint32_t texture_get_texid(RID p_texture) const;
+	Vector3i texture_get_dimensions(RID p_texture) const;
```

Comment:
```suggestion
	Vector3i texture_get_size(RID p_texture) const;
```
We use "size" everywhere else. So I would keep it as size here